robots.txt.sample

```diff
@@ -15,26 +15,38 @@
 
 User-agent: *
 
+## How many seconds a crawler should wait before loading and crawling page content
+## Set a custom crawl rate if you are experiencing traffic issues with your server
+## https://www.contentkingapp.com/academy/robotstxt/faq/crawl-delay-10/
+
+Crawl-delay: 10
+
 ############################################
 ## Allow to crawl paging (paging inside a listing with more params are disallowed below)
+
 Allow: /*?p=
 
 ############################################
-## Do not crawl non-SEF paths and generated content (if you use a shop id in URL, you must prefix with * or copy for each shop)
+## Do not crawl non-SEF paths and generated content (if you use a store id in URL you must prefix with * or copy for each store)
 
 Disallow: */index.php/
 Disallow: */catalog/product_compare/
 Disallow: */catalog/category/view/
 Disallow: */catalog/product/view/
 Disallow: */catalog/product/gallery/
-Disallow: */catalogsearch/
+Disallow: */catalogsearch/advanced
+Disallow: */catalogsearch/result
+Disallow: */catalogsearch/seo_sitemap
```

Comment:
this file is needed for google (and others) to easy find crawl-able URLs. Preventing them to crawl this removes the whole purpose of this endpoint.

Reply:
tested on 2 of my stores and that endpoint goes to 404... i'm not sure it exists

Reply:
which brings up the question, what links to it :/

Reply:
exactly, that's why I should stay `Disallow: */catalogsearch/` instead of the 3 split lines

Reply:
to make the change by removing those 4 lines is no problem. but keep in mind that so far OM has not provided a robots.txt file. who has Frontend links to / catalogsearch / seo_map and / catalogsearch / term / popular when using this file in its previous form with the only option `` `Disallow: / * / catalogsearch /` `` will remove the indexing of these two pages. obviously we are talking about bots that take into account the existence and content of this file.

informatively, I analyzed for a month all the bots that went through my websites and I can say that very few come in contact with robots.txt. GoogleBot ignores Crawl-delay.